deliver/lib/deliver/options.rb

```diff
@@ -135,6 +135,16 @@ def self.available_options
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_ITC_TEAM_NAME"] = value
                                      end),
+        FastlaneCore::ConfigItem.new(key: :short_team_id,
+                                     short_option: "-s",
+                                     env_name: "DELIVER_SHORT_TEAM_ID",
+                                     description: "The short ID of your team if you're in multiple teams, different from itc_team_id",
+                                     optional: true,
+                                     is_string: false, # as we also allow integers, which we convert to strings anyway
+                                     default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
```

Comment:
Actually, it's different. There's 2 different ids. When I ran the command mentioned in #1524 to find out what the `short_name` of the teams were, it didn't match with the ids given [here](https://github.com/fastlane/fastlane/blob/435f4847731be666e4f011f5093ab1a5331940c1/spaceship/lib/spaceship/tunes/tunes_client.rb#L79). From my investigation, it appears to be the developer team id rather than the iTunes Connect id.